src/slack/slack.js

```diff
@@ -239,9 +240,8 @@ export class Slack {
   showError(channel, ts, msg) {
     const text = `:exclamation: ${msg}`
     logger.warn(`showed error to user: ${text}`)
-    return this.apiCall(ts ? 'chat.update' : 'chat.postMessage', {
-      channel, ts, as_user: true, text, attachments: [],
-    })
+
+    return ts ? this.boltApp.client.chat.update({channel, ts, as_user: true, text, attachments: []}) : this.boltApp.client.chat.postMessage(params)
```

Comment:
I think the `params` here are a mistake. and you could do something like the previous code structure, check this:
```js
return this.boltApp.client.chat[ts ? 'update' : 'postMessage']({channel, ts, as_user: true, text, attachments: []})
```

Reply:
I think here I wanted to assign `const params = {channel, ts, as_user: true, text, attachments: []}` and put params in both places.
Changed as you suggested.